bin/build-openapi-yaml.rb

```diff
@@ -1,3 +1,17 @@
+# Copyright (c) 2025, FusionAuth, All Rights Reserved
```

Comment:
this includes the changes in https://github.com/FusionAuth/fusionauth-client-builder/pull/134

Reply:
those were merged into this branch via issue 1 branch.